setup.py

```diff
@@ -13,20 +13,26 @@
 requires = [r.strip() for r in req_data.split() if r.strip() != '']
 requires = list(reversed(requires))
 
+# testing dependencies
+req_data = open(os.path.join(__here__, 'test_requirements.txt')).read()
+tests_require = [r.strip() for r in req_data.split() if r.strip() != '']
+tests_require = list(reversed(tests_require))
+
 # temporary workaround for a Python 2 CherryPy bug, for which we opened a pull request:
 # https://bitbucket.org/cherrypy/cherrypy/pull-request/85/1285-python-2-now-accepts-both-bytestrings/
 if sys.version_info[0] == 2:
     requires = ['CherryPy==3.2.2' if 'cherrypy' in r.lower() else r for r in requires]
 
 if __name__ == '__main__':
+    setup_requires = {'setup_requires': ['distribute']} if sys.version_info[0] == 2 else {}
     setup(
         name=pkg_name,
         version=__version__,
         description='Sideboard plugin container.',
         license='BSD',
         scripts=[],
-        setup_requires=['distribute'],
         install_requires=requires,
+        tests_require=tests_require,
```

Comment:
I have no strong opinions on this, so whatever fits your needs best. Just wanted to point out that by separating out your test requirements, but leaving your tests in place, you are now shipping code that will break if it's imported.